lib/spack/spack/schema/spack.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+"""Schema for spack environment
+
+.. literalinclude:: _spack_root/lib/spack/spack/schema/spack.py
+   :lines: 20-
+"""
+from typing import Any, Dict
+
+from llnl.util.lang import union_dicts
+
+import spack.schema
+import spack.schema.gitlab_ci as ci_schema  # DEPRECATED
+import spack.schema.merged as merged_schema
```

Comment:
@becker33 @tldahlgren Is there any reason why we added this new schema, which seems a duplicate of `env.py`?

Reply:
I don't recall and haven't found it in my notes.